sklearn/datasets/tests/test_covtype.py

```diff
@@ -23,3 +23,22 @@ def test_fetch(fetch_covtype_fxt):
     # test return_X_y option
     fetch_func = partial(fetch_covtype_fxt)
     check_return_X_y(data1, fetch_func)
+
+
+def test_fetch_asframe(fetch_covtype_fxt):
+    pd = pytest.importorskip('pandas')
+    bunch = fetch_covtype_fxt(as_frame=True)
+    frame = bunch.frame
+    assert hasattr(bunch, frame) is True
+    assert frame.shape == (581012, 55)
+    assert isinstance(bunch.frame, pd.DataFrame)
```

Comment:
So the same here. It should be in the common test regarding the type checking. You can only check the shape of the data

Reply:
ok, will get rid of the type checking here.